src/shared/sam/localLambdaRunner.ts

```diff
@@ -319,9 +319,9 @@ export async function invokeLambdaFunction(
     if (!config.noDebug) {
         if (config.invokeTarget.target === 'api') {
             const payload = JSON.parse(await readFile(config.eventPayloadFile, { encoding: 'utf-8' }))
-            await requestLocalApi(ctx, config.api!, config.apiPort!, payload).finally(() => {
-                stopApi(config)
-            })
+            await requestLocalApi(ctx, config.api!, config.apiPort!, payload)
+            // Wait for cue messages ("Starting debugger" etc.).
+            await samStartApi
```

Comment:
this improves reliability of APIGW debugging, especially for cases where downloading an image takes a long time.